examples/text-editor/textEditor.wyv

```diff
@@ -106,6 +107,10 @@ def registerPlugin(plugin: Plugin): {RegisterPlugin, plugin.Run} Unit
   plugins.add(nativeActionCreator.createWithAction(plugin.getName(), () => plugin.run()))
   logger.updateLog("Registered plugin " + plugin.getName() + ".\n")
 
+def registerThemePlugin(plugin: ThemePlugin): {RegisterPlugin, plugin.Run} Unit
+  plugins.add(nativeActionCreator.createWithAction(plugin.getName(), () => plugin.run()))
+  logger.updateLog("Registered theme plugin " + plugin.getName() + ".\n")
+
```

Comment:
Due to the way Java works, theme plugins can only be run once, during the application setup. Thus, theme plugins shouldn't be registered but run instead. Please remove this method and create the `runThemePluginOnSetup` instead.